packages/utils/src/utils/agent.utils.ts

```diff
@@ -14,26 +15,21 @@ export const defaultAgent = (): Agent =>
 
 /**
  * Create an agent for a given identity
- * @param identity A mandatory identity to use for the agent
- * @param host An optional host to connect to
- * @param fetchRootKey Fetch root key for certificate validation during local development or on testnet
- * @param verifyQuerySignatures Check for signatures in the state tree signed by the node that replies to queries - i.e. certify responses.
- * @param retryTimes Set the number of retries the agent should perform before errorring.
+ *
+ * @param {CreateAgentParams} params The parameters to create a new HTTP agent
+ * @param {Identity} params.identity A mandatory identity to use for the agent
+ * @param {string} params.host An optional host to connect to
```

Comment:
Maybe add "Particularly useful for local development" ?

Reply:
Added